README.md

```diff
@@ -295,24 +295,25 @@ the GitHub release.
 
 The name for the ZIP must follow the following pattern:
 
-* `php_{extension-name}-{tag}-{php-maj/min}-{compiler}{"-nts"}?{-arch}.zip`
+* `php_{extension-name}-{tag}-{php-maj/min}-{ts|nts}-{compiler}-{arch}.zip`
+* Example: `php_xdebug-3.3.2-8.3-ts-vs16-x86_64.zip`
 
 The descriptions of these items:
 
 * `extension-name` the name of the extension, e.g. `xdebug`
 * `tag` for example `3.3.0alpha3` - defined by the tag/release you have made
 * `php-maj/min` - for example `8.3` for PHP 8.3.*
-* `compiler` - usually something like `vc16` - this should match
-  the `PHP_COMPILER_ID`
-* `-nts` - optional - non-thread safe. If omitted, thread-safe mode (ZTS) is
-  assumed.
-* `-arch` - for example `x86_64`, fetch using `php -r "echo php_uname('m');"`.
+* `compiler` - usually something like `vc6`, `vs16` - - fetch from
+  'PHP Extension Build' flags in `php -i`
+* `ts|nts` - Thread-safe or non-thread safe.
+* `arch` - for example `x86_64`, fetch using `php -r "echo php_uname('m');"` or
```

Comment:
`php_uname('m')` is apparently unreliable to determine the platform that PHP is compiled for; it appears to just echo `uname -m`; if you're running a 64-bit system, but have a 32-bit compiled PHP, then `php_uname('m')` will still return 64-bit.

After some discussion (thanks @alcaeus @derickr !), we can use `PHP_INT_SIZE` to determine 32-bit vs 64-bit.